tests/py/unit/test_tools_modify.py

```diff
@@ -7,5 +9,11 @@ def test_filter_by_label():
     film = bulk("Cu", cubic=True)
     interface_atoms = substrate + film
     interface_atoms.set_tags([1] * len(substrate) + [2] * len(film))
-    film_extracted = filter_by_label(interface_atoms, 2)
-    assert (film.symbols == film_extracted.symbols).all()
+    material_interface = Material(from_ase(interface_atoms))
+    film_extracted = filter_by_label(material_interface, 2)
+    film_material = Material(from_ase(film))
+
+    # Ids of filtered elements will be missing, comparing the resulting values
+    assert [el["value"] for el in film_material.basis["elements"]] == [
```

Comment:
The comparison needs be by for `atomic_values` class with `__eq__`

Reply:
A task for that created